changelogs/fragments/5108-proxmox-node-name-condition.yml

```diff
@@ -0,0 +1,2 @@
+deprecated_features:
```

Comment:
@felixfontein wouldn't this be a bug fix instead?

Reply:
Yes, it's fixed now, was a copy-paste error.

Reply:
Indeed, thanks for spotting that!